app/controllers/lessons_controller.rb

```diff
@@ -1,6 +1,7 @@
 class LessonsController < ApplicationController
   def show
-    @lesson = decorated_lesson
+    @lesson = Lesson.find(params[:id])
+    @course = @lesson.course
```

Comment:
Whereabouts do we need in the course ivar? I may have missed it, when looking through the views I only saw the course getting accessed directly from the lesson

Reply:
Good spot 💪